### PR TITLE
Remove wrong code from JarParser.java.

### PR DESCRIPTION
**Pull request by AxelBelin**
refactor JarParser.java to remove inappropriate code.

### CODE REVIEW

File: src/main/java/com/github/donnebelin/umldoc/classfile/JarParser.java [hunk 1]
```diff
@@ -44,12 +41,58 @@ public JarParser() throws IOException {
...
+
+    if ((access & Opcodes.ACC_ENUM) != 0) {
+      return Stereotype.ENUM;
+    }
+
+    if ((access & Opcodes.ACC_INTERFACE) != 0) {
+      return Stereotype.INTERFACE;
+    }
+
+    if ((access & Opcodes.ACC_ANNOTATION) != 0) {
```
Comment (forax) [inline src/main/java/com/github/donnebelin/umldoc/classfile/JarParser.java:57]: an annotation is an interface so the stereotype of an annotation will be "INTERFACE"

File: src/main/java/com/github/donnebelin/umldoc/classfile/JarParser.java [hunk 2]
```diff
@@ -44,12 +41,58 @@ public JarParser() throws IOException {
...
+      return Stereotype.ABSTRACT;
+    }
+
+    return Stereotype.CLASS;
+  }
+
+  private static Set<Modifier> modifiers(int access) {
+    var modifiers = new HashSet<Modifier>();
+
+    if (java.lang.reflect.Modifier.isStatic(access)) {
```
[the remark this answers is not in the document]
Comment (forax) [reply src/main/java/com/github/donnebelin/umldoc/classfile/JarParser.java:67]: This review is still on hold because of the comment above.

**Comment by forax** [comment]
I can not merge this PR because it is marked as Draft

**Comment by forax** [comment]
I still can not merge this pull request until you remove the draft state of this PR

**Comment by AxelBelin** [comment]
Add associations support and refactor JarParser.
Also add a new package for builders.

**Pull request merged**